src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -19,13 +20,77 @@ type HacbsApplicationDetailsProps = {
+            description: `Getting started with an app.`,
+            introduction: `You created an app and added a component to it. Good job! To get started using your app, follow the steps in this quick start guide.`,
+            tasks: [
+              {
+                title: `Add components and builds`,
+                description: `
+1. To add components and builds, go to the **Actions** dropdown and select "Add Component".
+2. Add the components you want included, then you will receive a PR to add the build pipeline to your code.
+3. View the Pull Requests we have sent you by clicking on the **Merge Pull Requests** button. You will be directed to the Pull Requests list.
+4. Customize this build pipeline according to your development needs.
+5. Merge to the PR to add the build pipeline to your code.`,
+              },
+              {
+                title: `Add integration test`,
+                description: `Integration test is a pipeline, set from GitHub, with the goal to test each component individually and then once more together as one application. Because it is unique per app, this task is on you. But we can help you get there. Check out our guide on how to set it up!  
+                \
+                [Learn about integration tests](http://www.redhat.com)
+1. To add an integration test pipeline, go to the **Action** dropdown of your application and select "Add integration test".
+2. In the URL line, add a URL to your integration test pipeline.
+3. To ensure your app is only released when the test passes, mark the integration test as **Mandatory for Release**.`,
+              },
+              {
+                title: `Manage environments`,
+                description: `Now, we need to deploy your builds somewhere. Add a static environment, an environment available only on this workspace. This will be your development environment.  
+                [Learn more about Static Environments](http://www.redhat.com)
+1. To add a static environment, go to the **Actions** dropdown of your application and select **Add environment**. 
+2. On the next page, select a static environment.
+3. On the Create Static environment page, complete all the mandatory input.
+4. Click **Create**.`,
+              },
+            ],
+            conclusion: `Congratulations! You're all set up.
+            <br><br>
+            Now it's time to:  
+              * Connect to a managed environment to release your app to  
+              * Push some code, monitor its process
+            <br><br>
+            If you get lost, this guide is here to help. You can access it at any time from the **Actions dropdown**.`,
+          },
+        },
+      ]);
+      quickStarts.toggle('hacbs-gs-apps');
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
When disabling an eslint rule, add a comment as to why.

Reply:
added a comment, i do not believe there are comments accompanying any other disabled lint rules in hac-dev

Reply:
It's a convention we try to impose when we can because disabling eslint rules is bad. Obviously not everyone will follow the guidelines and not everyone will enforce them.